bff-service/src/app.service.ts

```diff
@@ -0,0 +1,57 @@
+import { HttpService, HttpStatus, Injectable, Request } from '@nestjs/common';
+import { config } from 'dotenv';
+import { catchError, map } from 'rxjs/operators';
+import { of } from 'rxjs';
+
+config();
+
+@Injectable()
+export class AppService {
+  constructor(private httpService: HttpService) {}
+  bff(req: Request) {
+    console.log('url', req.url);
+    console.log('method', req.method);
+    console.log('body', req.body);
+
+    const recipient = req.url.split('/')[1];
```

Comment:
Seams like recipient is used as the part of URL rather then name of service, this makes impossible to make requests to /api/auth/login (which present in cart service) only profile links are available